layouts/community/ergodox/osx_neo2/layers.h

```diff
@@ -0,0 +1,7 @@
+#define NEO_1   0      // layer_0
```

Comment:
Turning these defines into an `enum` will make these definitions easier to manage.

```c
enum layers {
    NEO_1,      // layer_0
    NEO_3,      // layer_1
    NEO_4,      // layer_2
    NEO_5,      // layer_3
    NEO_6,      // layer_4
    US_1,       // layer_5
    FKEYS       // layer_6
};
```

This is programmatically identical to your definitions, but doesn't require any manual defining.

Reply:
Done